Makefile

```diff
@@ -13,7 +13,7 @@ migrate: build
 shell: build
 	docker-compose run web python manage.py shell
 
-env:
+cpenv:
```

Comment:
When I run `make env` it does some thing super weird like it's a default command in make.